app/controllers/EntryDetailsController.scala

```diff
@@ -44,6 +44,13 @@ class EntryDetailsController @Inject()(
 
   val form = formProvider()
 
+  private def getBackLink(mode: Mode, userAnswers: UserAnswers): Call = {
```

Comment:
Could this be refactored? Maybe match on the `contains` and check if true or false?